eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,4 +1,5 @@
 variables:
   PythonVersion: '3.6'
   skipComponentGovernanceDetection: true
-  AzureSDKCondaChannel: https://azuresdkconda.blob.core.windows.net/channel1/
\ No newline at end of file
+  AzureSDKCondaChannel: https://azuresdkconda.blob.core.windows.net/channel1/
+  Package.EnableSBOMSigning: true
```

Comment:
Do we need to set this yet? Will it even work for public builds?

Reply:
Either way, it won't do any harm! (At least not according to their documentation)

I think that at the outset they won't enforce on this, but they _will_. Leaving it on by default is good imo.